src/projects.md

```diff
@@ -1,23 +1,102 @@
+ - [Active projects](#ongoing-projects-that-support-openoakland-s-operation)
+ - [Inactive projects](#inactive-projects)
+ - [Decommissioned projects](#decommissioned-projects)
+ - [Becoming an OpenOakland project](#becoming-an-openoakland-project)
+ - [Providing feedback](#providing-feedback)
 
 ---
 
-<div id="projects">
-  <h2>Active Projects</h2>
-  {% for project in site.data.active_projects %}
-    {% include project.html %}
-  {% endfor %}
-  <hr/>
-  <h2>Inactive Projects</h2>
-  <p><em>These projects are some of OpenOakland’s original projects. They are no longer actively being developed and aren’t accepting volunteers.</em></p>
-  {% for project in site.data.inactive_projects %}
-    {% include project.html %}
-  {% endfor %}
+
+{% include linkedHeading.html heading="Ongoing projects that support OpenOakland's operation" level=2 %}
+
+These projects are a great way to get your feet wet as you get to know OpenOakland.
+
+<div id="projects-openoakland">
+{% for project in site.data.openoakland_projects %}
+{% include project.html %}
+{% endfor %}   
 </div>
+
+---
+
+{% include linkedHeading.html heading="Active projects" level=2 %}
+
+The following projects are official OpenOakland projects that are being actively supported by an existing project team. If you see something that interests you, there are a couple of ways you can get involved:
+
+- Join us for our Tuesday [Hack Nights](https://www.meetup.com/OpenOakland/events/) to connect with the project team.
+- Join our [Slack workspace](http://slack.openoakland.org/) and introduce yourself in the project's channel listed in the description.
+
+<div id="projects-active">
+{% for project in site.data.active_projects %}
+{% include project.html %}
+{% endfor %}
+</div>
+
+---
+
+{% include linkedHeading.html heading="Inactive projects" level=2 %}
+
+The following projects aren’t currently being worked on by a project team and are considered dormant. If you're interested in resuming one of these projects, you'll need to start by updating or completing a new [Project Brief](https://docs.google.com/document/d/1k24P9JiAUEzJLPFRDjVh7aRZexax6NUhfPFLSI3R80M/edit?usp=sharing) (most of these projects were developed prior to our current project development process and will need to go through the new process to resume).
```

Comment:
In my experience, terms like `current` and `new` age poorly in documentation. Also, if this is someone's first interaction with OpenOakland, they may want to know we're a helpful group. Finally, repeating `develop` may feel repetitive ;P Perhaps...
>(most of these were created before we implemented our project development process.) Please reach out if you want help in completing the brief and resuming the project.